websites.yml

```diff
@@ -7,6 +7,7 @@ urls:
   opensuse: https://connect.{w}.org/pg/profile/{u}
   pinterest: https://in.{w}.com/{u}
   tumblr: https://{u}.{w}.com
+  asciinema: https://{w}.org/~{u}
```

Comment:
you should put in alphabetic order. and modify below ones too.

Reply:
@margobra8 pls fix this